ci-operator/config/openshift/kedacore-keda/openshift-kedacore-keda-main.yaml

```diff
@@ -44,6 +70,58 @@ tests:
     clientset-generate
   container:
     from: src
+- always_run: false
```

Comment:
why false?  should this be paired with a run_if_changed spec?

Reply:
Oh hey, good catch. That was me originally being paranoid because I wasn't sure how stable this was going to be, but this seems decently stable. 

I've removed that and added a `skip_if_only_changed` to exclude some of our doc (it was easier to exclude than include since so little was excluded).